ui/src/schemas/index.ts

```diff
@@ -69,3 +84,10 @@ export const arrayOfTelegrafs = [telegraf]
 
 export const scraper = new schema.Entity(ResourceType.Scrapers)
 export const arrayOfScrapers = [scraper]
+
+export const addLabels = <R extends {labels?: Label[]}>(resource: R): R => {
```

Comment:
ooh- do resource types come out of this function with required labels properties? I wonder if the last `R` on this line should be `R & {labels: Label[]}` instead?

Reply:
Separately- should we not be adding labels to buckets?

Reply:
There's also a version of this in `src/labels/utils/index.ts` should this replace it?

Reply:
I'll be doing a whole labels overhaul during the `Label` normalization epic!